### DISCUSSION TITLE
Choosing Hardware for LTSP server with 50 thin clients

### DISCUSSION THREAD

**Discussion by linuxacademyir** [ltsp/ltsp#237] [Q&A]
Hello
Thank you for your great LTSP, its has been a great help in our company. 
we have a call center with about 16 clients, running LTSP5, handling VOIP calls using linphone which is a SIP phone running as localapps, coz  running the linphone, from server gives us an error "there is already a version of linphone running". thats why we had to use ltsp-locallapps.
in Short all clients are using some web based CRMs and a SIP client.

The current thin clients are HP with 2 and 4 G ram and cpu with 500 benchmark, the server cpu is corei 3, with 16 G ram.  Some times the cpu is at 100 % usage and ram is at 70 % usage so clients freeze. 

We are expanding to 50 clients. planing to buy a new server. So here are the questions:
1) With  current Thin clients should we stay with LTSP5 or we can move to LTSP9?
2) Since the LTSP9 dosnt support localapps, will we have problem running the linphone from server?
3) I am confused about choosing the new server hardware. i have 2 options:
- one is Intel Corei 9900k CPU (16 MB cache/3.60 Ghz/8cores) with ASUS maximus XI Hero as a mother board with 16 G of Ram 
- second one is HP server with Xeon E5-2690(30 cache/2.60 Ghz/12cores) V3 with 32 G ram 
so the question is now, which one is better choice for LTSP, does LTSP need more Cores/cache or more Frequency "GHz". 

Thanks in advanced.

**Comment by enaut** [comment]
LTSP19 generally is rather a FAT-client setup. Meaning the client computers do a lot more and the server only provides the operating system. So if you'd upgrade to ltsp19 youd'd want better client hardware and the serverhardware can probably stay rather low end.

Also see: https://ltsp.org/docs/preparation/ (third paragraph)

**Comment by linuxacademyir** [comment]
thank you for your reply, so with current client hardware you recommend to go with LTSP5. So how about the Server requirements if i go with LTSP5 with 50 Thin clients? i would appreciate any advises

**Comment by alkisg** [comment]
Running ltsp5 localapps is about the same as running ltsp20 fat clients, as in both cases the apps run on the clients.
A beefy server would only be needed if you had a thin client setup, which you don't.

I would suggest to install Ubuntu 18.04 MATE or Lubuntu with ltsp20 and test with your existing clients, if it works well enough for you or not.

The specs for localapps and for fat clients are about the same as for standalone clients, i.e. the standard Ubuntu specs.

**Comment by linuxacademyir** [comment]
thank your for your answer, 
so if the LTSP20 works with my current set up, i dont need a heavy server?
and just generally asking, which CPU is the best for LTSP? Xeon CPUs or Corei CPUs?
we are currently using ubuntu Mate. how about Ubuntu Mint, which of these 3 are the best choice for LTSP?

**Comment by alkisg** [comment]
LTSP doesn't have any specific requirements. Just google "which is the best CPU for Linux".
Also, "Xeon" or "Core i" or "number of cores" doesn't really describe the CPU.
Use the cpubenchmark.net web site to see the score for the specific CPU you're asking.
There are Xeon CPUs with 1000 score and there are Core i CPUs with 10000 score (ten times faster), **and** the opposite.

But again, the server CPU doesn't matter, only the client CPU matters.

If you're already using Ubuntu MATE, it's great, continue using it.

**Comment by linuxacademyir** [comment]
hello thank you 
i finally chose the xeon processor as it seems to be a better option for a server. 
i tried your solution with my current thin clients, they are not good for LTSP 19. 
the clients are too slow. 
so i tired to install the LTSP5 then, using both the  chroot and chrootless. in both method. the server DHCP works fine, client receives an IP. tries to boot up. shows some OK. but at the very last stage, it gives an error. 
squashfs_read_data failed to read block xxxx. 
can you please advice, i really need to make the LTSP5 work to be able to use my old thin clients.

**Comment by alkisg** [comment]
Hi @mehdizeus,

this github community site is for the new LTSP;
the old LTSP5 is supported in the sourceforge mailing lists:
https://sourceforge.net/p/ltsp/mailman/

Try asking there!
